ControlRoomApplication/ControlRoomApplication/Controllers/RadioTelescopeControllers/RadioTelescopeController.cs

```diff
@@ -792,7 +796,7 @@ private void SensorMonitor()
                 elTempSafe = checkTemp(elTemp, elTempSafe);
 
                 // Determines if the telescope is in a safe state
-                if (azTempSafe && elTempSafe) AllSensorsSafe = true;
+                if (azTempSafe && elTempSafe && CompareMotorAndAbsoluteEncoders(currentMotorPosition, currentABSPosition)) AllSensorsSafe = true;
```

Comment:
We also need to check if the simulation sensor network is active. If it is active, then the absolute orientation will constantly be changing to random values. You can check if the simulation sensor network is null, like how I check it for software stops. If it isn't null, always return true because that means we are running a simulation.

I initially thought of sticking the check in the `CompareMotorAndAbsolueEncoders()` method but it would mess up the unit test.

- [ ] Add check for simulation sensor network